dietpi/dietpi-cleaner

```diff
@@ -24,9 +24,12 @@
 	G_INIT
 	#Import DietPi-Globals ---------------------------------------------------------------
 
+	#Exit trap (dietpi-globals)
+	G_EXIT_CUSTOM(){ unset aEnabledCleaners; }
```

Comment:
@Fourdee 
However, I am still not sure if this has any effect or all allocated RAM usage is anyway freed, when the script exits.

Reply:
@Fourdee 
Jep seems like. I mean also even large arrays take a tiny amount of RAM, compared to the whole shell, but especially in large scripts where variables might be used doubled (e.g. dietpi-software/config) I would consider it still good practice to unset any used variable, after used.